spinalcordtoolbox/scripts/sct_compute_compression.py

```diff
@@ -500,14 +501,12 @@ def save_df_to_csv(dataframe, fname_out):
     :return:
     """
     if os.path.isfile(fname_out):
-        # Concatenate existing CSV file to the new CSV file
-        dataframe_old = pd.read_csv(fname_out)
-        dataframe = pd.concat([dataframe_old, dataframe], axis=0, ignore_index=True)
-        # Merge rows that have the same 'filename', 'compression_level', and 'slice(I->S)'
-        dataframe = dataframe.groupby(
-            [dataframe.iloc[:, 0], dataframe.iloc[:, 1], dataframe.iloc[:, 2]]
-        ).max()  # Use max for a tiebreaker (this should never happen unless the user computes the same metric twice)
-    dataframe.to_csv(fname_out, na_rep='n/a', index=False)
+        # Combine the data with the existing CSV file.
+        # Rows with the same (filename, compression_level, slice) triple are merged together.
+        # Metric values from the new dataframe take priority over old CSV file values.
+        dataframe_old = pd.read_csv(fname_out, index_col=INDEX_COLUMNS)
+        dataframe = dataframe.combine_first(dataframe_old)
```

Comment:
From what I understand, `combine_first` is basically like [`combine`](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.combine.html#pandas.DataFrame.combine), but without the need to specify a tiebreaker function, since the original dataframe will be prioritized every time? Nice!

Reply:
Yup! Having both `combine` and `combine_first` available is very convenient.